packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-node.vue

```diff
@@ -87,15 +93,30 @@ const getStatus = async (runId: string) => {
 	return pollerResults;
 };
 
+function startTimer() {
+	clearTimeout(timer.value);
+	timer.value = setTimeout(
+		() => {
+			const state = _.cloneDeep(props.node.state);
+			state.message = "Process is stuck in Funman, click 'Stop' to cancel";
```

Comment:
The `click stop to cancel` doesn't work on the node as a message.
I would prefer the whole system to not be a bunch of props to pass a message around, but instead a boolean prop that mentionned that it is stuck. Then each the drilldown preview and the node can each convey that information the best for them.